docs/SPEC.md

```diff
@@ -395,10 +393,16 @@ The `task` option is used by a scheduler to schedule the execution of a Flux que
 
 The `location` option is used to set the default time zone of all times in the script.
 The location maps the UTC offset in use at that location for a given time.
-The default value is set using the time zone of the running process.
+The default value is UTC.
+The value is specified either as the name of a time zone in the IANA Time Zone database or as a fixed offset.
+If the string value is in the format `(+|-)HH:MM` then it will be interpreted as a fixed zone offset.
+
+    option location = "-05:00" // set location to a fixed offset 5 hours west of UTC
+    option location = "America/Denver" // set location to be America/Denver
 
-    option location = fixedZone(offset:-5h) // set timezone to be 5 hours west of UTC
-    option location = loadLocation(name:"America/Denver") // set location to be America/Denver
```

Comment:
Why the change to use a string instead of an object?

Reply:
I don't think we can represent it as an object. We cannot include the `time.Location` object which is really the only important part. So I opted for a string out of the available types.